### ISSUE TITLE
slow

### ISSUE THREAD

**Issue by turk-jk** [mapo80/SiriWave#1]
the animation is super slow

**Comment by mapo80** [comment]
Yes, Is true. You can increase speed changing speed property value. 
Have you tried?

**Comment by turk-jk** [comment]
I tried the speed but no luck however it works great when there's mic recording which serve my purpose. 

I am trying to init the view programmatically. but it just doesn't work (without storyboard)
in other work if I init the view with frame
in `viewDidLoad` or `viewWillAppear` or `viewDidAppear `
like this
`siriWave = SiriWaveView(frame: CGRect(x: 0, y: 100, width: self.view.frame.width, height: 300))`
I am not sure what I am missing

**Comment by oveappnap** [comment]
I added a button through which the animation starts then again after press the button I set the ampler and speed to 0 so it returns into the first state 

but after pressing the button again it starts animating super quickly. how to prevent the 2nd scenario?
